cli/standalone.rs

```diff
@@ -109,6 +109,16 @@ fn read_string_slice(
 
 const SPECIFIER: &str = "file://$deno$/bundle.js";
 
+fn module_supported(specifier: &str) -> Result<(), AnyError> {
+  if specifier == SPECIFIER || specifier.starts_with("data:text/javascript;") {
```

Comment:
Idk if this necessary but maybe change the specifier starts with to `"data:text/javascript;base64,"`.

Reply:
Specifiers can also start with ` data:text/javascript;charset=utf-8` IIRC

Reply:
it should be enough though, since it's starting with...